packages/dashboard/src/services/accountsApi.js

```diff
@@ -1,5 +1,3 @@
 import ky from "ky";
 
-const prefix = process.env.NEXT_PUBLIC_PORTAL_DOMAIN ? `https://account.${process.env.NEXT_PUBLIC_PORTAL_DOMAIN}` : "";
-
-export default ky.create({ prefixUrl: `${prefix}/api` });
```

Comment:
removed prefix because when running dashboard from server domain, requests were hitting portal domain and cookie did not authenticate them, we need them to hit the domain that we are currently on